### DISCUSSION TITLE
posts/movie_streaming

### DISCUSSION THREAD

**Discussion by giscus** [whexy/whexy-blog-comments#18] [General]
# posts/movie_streaming

今天得空，介绍上个月的成果——一套宿舍电影放映系统的构建。寝室里有一台大屏显示器，晚上关了灯，连上蓝牙低音炮，大家围坐一起看一部电影，真是美滋滋的体验~ 床上有一个支架，用iPad连接AirPods，晚上拉上遮光帘一个人欣赏一部电影，也是美滋滋的体验…

https://www.whexy.com/posts/movie_streaming

**Comment by hi-nikola** [comment]
so cool .很有意思。上学期间一直听到各种南科大传闻，总觉得是充满神秘色彩。很喜欢博主搭建的博客

**Comment by Hou-Xiaoxuan** [comment]
> 你确定在校外的大好时光不用来和妹子吃饭聊天玩耍？你们为啥不去电影院？

> 我还是推荐各位去电影院，否则你可能会赢了面子，输了人生。
???感觉遭到了攻击
